src/styles/catalog.scss

```
@@ -0,0 +1,19 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px calc((100% - #{$catalog-width}) / 2);
```

Better calculate (without calc-func) container's max-width. This solution it's not good 
```suggestion
  padding: 50px 0;
```